packages/pigeon/lib/generator_tools.dart

```diff
@@ -8,7 +8,7 @@ import 'dart:mirrors';
 import 'ast.dart';
 
 /// The current version of pigeon. This must match the version in pubspec.yaml.
```

Comment:
Do the pigeon automated tests check this in CI? If not, that would be a good thing to implement.

Reply:
Yep, we have a test for that https://github.com/gaaclarke/packages/blob/pigeon-with-analyzer/packages/pigeon/test/version_test.dart#L11:L11